lib/nanoc/cli/commands/create-site.rb

```diff
@@ -281,7 +282,7 @@ def run
 
       # Check whether site exists
       if File.exist?(path)
-        raise Nanoc::Int::Errors::GenericTrivial, "A site at '#{path}' already exists."
```

Comment:
The error message isn’t as clear as it could be. Suggestion:

> The site was not created, because #{path} already exists. If you really want to create a site in this directory, re-run this command with `--force`.